go.mod

```diff
@@ -13,60 +13,61 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware/providers/zerolog/v2 v2.0.0-rc.2
 	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.0.0-rc.2
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/klauspost/compress v1.15.10
-	github.com/labstack/echo/v4 v4.6.1
-	github.com/onflow/cadence v0.31.5
-	github.com/onflow/flow-go-sdk v0.31.3
+	github.com/klauspost/compress v1.15.13
+	github.com/onflow/cadence v0.38.0
+	github.com/onflow/flow-go-sdk v0.40.0
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/tsdb v0.7.1
-	github.com/rs/zerolog v1.28.0
+	github.com/rs/zerolog v1.29.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.0
-	github.com/ziflex/lecho/v2 v2.5.1
-	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
-	google.golang.org/api v0.85.0
-	google.golang.org/grpc v1.47.0
+	github.com/stretchr/testify v1.8.2
+	golang.org/x/sync v0.1.0
+	google.golang.org/api v0.114.0
+	google.golang.org/grpc v1.53.0
 )
 
 require (
-	github.com/onflow/flow-go v0.29.17-read-leaf-nodes
-	github.com/onflow/flow-go/crypto v0.24.4
-	github.com/optakt/flow-dps-rosetta v1.0.0
+	github.com/onflow/flow-go v0.30.1
```

Comment:
we need to upgrade to `v0.30.3-fix-read-leaf-nodes`, as [this PR did](https://github.com/onflow/flow-archive/pull/96/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R30)